packages/downloads/package.json

```diff
@@ -19,10 +19,12 @@
   "peerDependencies": {
     "@esri/arcgis-rest-feature-layer": "^2.21.0",
     "@esri/arcgis-rest-portal": "^2.21.0",
-    "@esri/arcgis-rest-request": "^2.21.0"
+    "@esri/arcgis-rest-request": "^2.21.0",
+    "@esri/hub-common": "^7.1.1"
   },
   "devDependencies": {
     "@esri/arcgis-rest-auth": "^2.21.0",
+    "@esri/hub-common": "^7.1.1",
```

Comment:
Does this need to be in both dev and peer dependencies?

Reply:
It's unclear to me, but that seems to be a pattern across several packages